### DISCUSSION TITLE
snippets/binary-search

### DISCUSSION THREAD

**Discussion by giscus** [thutasann/thuta-portfolio#29] [Announcements]
# snippets/binary-search

I'm a developer that loves building NEXTJS, ANGULAR & SpringBoot and IOS Mobile apps. Also loves building UI/UX with TailwindCSS, Material UI, RadixUI, etc

https://thutadev.vercel.app/snippets/binary-search

**Comment by thutasann** [comment]
if any discussion, pls lmk.
